sources/packages/api/src/esdc-integration/e-cert-integration/models/e-cert-integration-model.ts

```diff
@@ -1,5 +1,12 @@
-import { DisbursementValue, RelationshipStatus } from "../../database/entities";
+import {
+  DisbursementValue,
+  RelationshipStatus,
+} from "../../../database/entities";
 
+export const ECERT_SENT_TITLE = "ENTITLEMENT";
+export const CSGD = "CSGD";
```

Comment:
Please add some comments explaining what is each one, if we have this information.

Reply:
Please ensure that the comments are added as below to allow the IntelliSense to work.
```typescript
/**
 * Amount of Grant for Part-time Studies (CSGP-PT) at the study start.
 */
export const CSGD = "CSGD";
```